envFiles/.env.production

```
@@ -19,14 +19,18 @@ API_HOST=0.0.0.0
 API_IS_APPLY_DRIZZLE_MIGRATIONS=true
 API_IS_GRAPHIQL=false
 API_IS_PINO_PRETTY=false
+# Recommended for production (HTTPS): use secure cookies.
+API_IS_SECURE_COOKIES=true
 API_JWT_EXPIRES_IN=900000
 API_REFRESH_TOKEN_EXPIRES_IN=604800000
 API_JWT_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_JWT_SECRET key should go before the API_REFRESH_TOKEN_EXPIRES_IN key

(UnorderedKey)